README.md

```diff
@@ -34,27 +34,51 @@ Create a report object in your {{ application }}/reports.py. This should look so
 class YourFirstReport(Report):
     slug = "your-first-report"
     display_name = "Your First Report"
-    description = """Everyone has to have one report that is their first, this one is yours."""
-
-    def generate_report_data(self, *args, **kwargs):
-        return [
-            ReportFile(
-                file_name="some_file.txt", file_data=[['hello']]
-            )
-        ]
+    description = """Everyone has to have one report that is their first, this ones yours"""
```

Comment:
I think this `ones` a step backwards in terms of grammar unless we add an apostrophe?